src/test/java/io/openliberty/tools/intellij/it/SingleModJakartaLSTestCommon.java

```diff
@@ -129,8 +129,8 @@ public void testJakartaDiagnosticsInJavaPart() {
     /**
      * Tests Jakarta Language Server quick fix support in a Java source file
      */
-    @Test
-    @Video
+//    @Test
+//    @Video
```

Comment:
If you haven't already done so, we should have an issue open to restore this test so we don't lose track that we disabled it here.

Reply:
I was thinking we should reopen the issue that Trevor closed yesterday and add the note to re-add this test.